quote-server/quote-generator/src/RandomQuote.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+
+const RandomQuote = () => {
+  const [quote, setQuote] = useState("");
```

Comment:
In general we try to keep state having the same _type_ of value, so I'd recommend either initialising this to `null` rather than the empty string here.

Can you think about why this may be useful?